modules/ROOT/pages/installation/requirements.adoc

```diff
@@ -103,19 +108,15 @@ For cloud environments, and server-based, on-premise environments:
 | *Amazon Linux 2022 AMI, Amazon Linux 2023 AMI*   | Amazon Corretto 17, and OracleJDK 17
 | *CentOS Stream 8, 9*                             | OpenJDK 17, OracleJDK 17, and ZuluJDK 17
 | *Debian 11, 12*                                  | OpenJDK 17, OracleJDK 17, and ZuluJDK 17
-| *Red Hat Enterprise Linux Server 8.6, 8.8, 9.0, 9.2*  | Red Hat OpenJDK 17,  Oracle JDK 17, and ZuluJDK 17
-| *Ubuntu Server 16.04, 18.04, 20.04, 22.04*       | OpenJDK 17, OracleJDK 17, and ZuluJDK 17
-| *Windows Server 2016, 2019, 2022*                | OracleJDK 17, ZuluJDK 17
+| *Red Hat Enterprise Linux Server 8.10, 9.2, 9.4* | Red Hat OpenJDK 17,  Oracle JDK 17, and ZuluJDK 17
+| *Ubuntu Server 16.04, 18.04, 20.04, 22.04, 24.04*| OpenJDK 17, OracleJDK 17, and ZuluJDK 17
```

Comment:
Deprecated or not, I think we should remove ubuntu 16.04 and 18.04 from this list since they are not even supported by Ubuntu now without a "pro" license [source](https://ubuntu.com/about/release-cycle).
What do you think @b33f ?

Reply:
We can't arbitrarily just remove OSes that haven't been announced as deprecated.  This will upset customers and will have some strong feedback.